utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -224,7 +224,7 @@ def host_install_destdir(self, host_target):
                 # If this is one of the hosts we should lipo,
                 # install in to a temporary subdirectory.
                 return '%s/intermediate-install/%s' % \
-                    (self.args.install_destdir, host_target)
+                    (os.path.dirname(self.build_dir), host_target)
```

Comment:
Heh, I had a fix for this in another pull, #36917.

Reply:
Hah, I wish I'd seen that before! It's totally the fix for my problem, too